src/app_service/service/token/service.nim

```diff
@@ -137,8 +241,23 @@ QtObject:
       error "Tokens init error", errDesription = e.msg
 
   proc init*(self: Service) =
+    if(not main_constants.WALLET_ENABLED):
```

Comment:
Is this still needed?

Reply:
in case we bring back the the setting at some point, it could help having this functionality already present.